src/js/shield_defs.js

```diff
@@ -883,6 +883,19 @@ export function loadShields(shieldImages) {
     Color.shields.white
   );
 
+  // Mexico
+
+  // Carreteras Federales
+  shields["MX:MX"] = {
+    backgroundImage: shieldImages.shield_mx_mx,
```

Comment:
>  I've noticed that the 3-digit Mexico shields are the same width as the 2 digit versions in real world shields.

I think it’s entirely reasonable to depart from the literal sign metrics to make numbers a little more readable when the shape can accommodate that kind of stretching. But I think in this case it’ll work out OK for the normal three-digit routes. GUA 10D would be a different story, but that sounds like a special case.